typings.json

```diff
@@ -6,10 +6,13 @@
   },
   "globalDependencies": {
     "body-parser": "registry:dt/body-parser#0.0.0+20160619023215",
+    "cookies": "registry:dt/cookies#0.5.1+20160316171810",
```

Comment:
Where do we use this? I can't find any reference in the code. Is this something that Koa requires?